### PR TITLE
{vis}[iccifort/2020.4.304] SMV 6.7.17 (smokeview)

### PR DESCRIPTION
**Pull request by deniskristak**

### CODE REVIEW

File: easybuild/easyconfigs/s/SMV/SMV-6.7.17-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+local_buildopts += ' CFLAGS="-Dpp_LINUX" '
+
+prebuildopts = "cd %(builddir)s/smv-SMV%(version)s/Build/LIBS/intel_linux_64/ && "
+prebuildopts += " make all -f make_LIBS.make "
+prebuildopts += "SRCDIR=%(builddir)s/smv-SMV%(version)s/Source/ LIBDIR=%(installdir)s && "
+prebuildopts += "cd %(builddir)s/smv-SMV%(version)s/Build/smokeview/ && "
+prebuildopts += "export IFORT_COMPILER_LIB=$EBROOTICCIFORT%s && " % local_compiler_lib_append
+
+buildopts = ' intel_linux_64 ' + local_buildopts
+buildopts += ' && make -j 36 intel_linux_64_db ' + local_buildopts
```
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/s/SMV/SMV-6.7.17-intel-2020b.eb]: @deniskristak looking at the Makefiles, both `intel_linux_64` and `intel_linux_64_db` seem to be doing the same build, except the latter adds debug symbols, checks, warnings, etc.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3118.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a30a65d5439a25b3643cbcda47fecee8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3519.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/9451a710acb785c0abd8631876d66400 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @deniskristak!

**Pull request merged**